nestkernel/simulation_manager.h

```diff
@@ -26,6 +26,8 @@
 // C++ includes:
 #include <vector>
 
+#include <sys/time.h>
```

Comment:
Please move this above the C++ includes as in [stopwatch.h](https://github.com/nest/nest-simulator/blob/8adec3cd2bb8025ca07c91f8e281ecb71f13658f/libnestutil/stopwatch.h#L27)